package.json

```diff
@@ -28,6 +28,7 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^1.13.0",
     "@typescript-eslint/parser": "^1.13.0",
+    "@vue/compiler-sfc": "^3.0.0-alpha.1",
```

Comment:
do you need to explicity depend on this?

Reply:
Yes, remove it and you’ll see the warning.